core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -173,14 +174,25 @@ class LogCleaner(initialConfig: CleanerConfig,
     }
   }
 
+  /**
+   * Remove metrics when shutdown cleaner threads
```

Comment:
I find this sentence a bit confusing, I think keeping it short is more than enough:
```suggestion
   * Remove metrics
```